cpp/modmesh/math/Complex.hpp

```diff
@@ -121,13 +135,21 @@ struct ComplexImpl
 template <typename T>
 bool operator<(const ComplexImpl<T> & lhs, const ComplexImpl<T> & rhs)
 {
-    return lhs.norm() < rhs.norm();
+    if (lhs.real_v < rhs.real_v)
```

Comment:
Always use curly braces `{}` for a single-line block.

Comparison of complex numbers are trick.  Perhaps we should not provide the greater than and the less than operators.  We'll see.  Please leave a comment to document the trickiness.